packages/event-handler/src/rest/Router.ts

```diff
@@ -551,6 +551,43 @@ class Router {
       handler
     );
   }
+
+  /**
+   * Merges the routes, context and middleware from the passed router instance into this router instance
+   *
+   * @example
+   * ```typescript
+   * import { Router } from '@aws-lambda-powertools/event-handler/experimental-rest';
+   *
+   * const todosRouter = new Router();
+   *
+   * todosRouter.get('/todos', async () => {
+   *   // List API
+   * });
+   *
+   * todosRouter.get('/todos/{todoId}', async () => {
+   *   // Get API
+   * });
+   *
+   * const app = new Router();
+   * app.includeRouter(todosRouter);
+   *
+   * export const handler = async (event: unknown, context: Context) => {
+   *   return app.resolve(event, context);
+   * };
+   * ```
+   * @param router - The Router from which to merge the routes, context and middleware
+   * @param options.prefix - An optional prefix to be added to the paths defined in the router
```

Comment:
I think we need a line for `@param options` alone above, or it won't have anything before you destructure it (same above)